### DISCUSSION TITLE
Edge model: comments received

### DISCUSSION THREAD

**Discussion by Kevsy** [camaraproject/EdgeCloud#232] [General]
From Andreas Florath

•	No concentration on Telco Operators (so IMHO no concept of Operator Platform or similar is needed).
•	Technology / runtime independent. The technology cycle for VMs is already on the way down. Can anybody tell if we are still using Container in 5-8 years?
•	UE is highly Telco Operator dependent and will not be used in this way in a generic industry platform.
•	Also, we should discuss concepts like cloudlets, flavor, edge cloud, mec platform as they already point to some underlying technologies and limit a generic solution.
•	In addition, there is IMHO the need start collecting other expected parameters, like number of requests per second, reliability or high availability. (My idea is that we will end up having several thousand requests a second which is too much using a single endpoint for many resources.)

**Comment by ThomasEdgeXR** [comment]
Point 1: The CAMARA project charter specifically limits the scope to Telco APIs (https://github.com/camaraproject/Governance/blob/main/ProjectCharter.md#scope-of-the-project) 
Point 2:Not clear to me what the proposal is. The APIs need to cover different workload types that are relevant today as the APIs should be used today.
Point 3: Same as 1
Point 4: Unclear what is the proposal here
Point 5: Generally agreed but unclear to me how that relates to the API design, i.e. whether those parameters will have any influence on the actual APIs. Would be good to get more details on the actual proposal.
